app/templates/api.ejs

```diff
@@ -0,0 +1,7 @@
+<%- include("includes/header.ejs") %>

```

Comment:
Seems like these template files could be generalized into just one.

Reply:
Dev mode toggle has now been added to the config file. It bypasses the API key check when making requests.